test/cypress/e2e/registration.spec.js

```diff
@@ -0,0 +1,118 @@
+import networks from '../../constants/networks';
+
+const ss = {
+  networkDropdown: '.network',
+  networkStatus: '.network-status',
+  newAccountBtn: '.new-account-button',
+  nodeAddress: '.peer',
+  errorPopup: '.toast',
+};
+
+/**
+ * Generates a sequence of random pairs of x,y coordinates on the screen that simulates
+ * the movement of mouse to produce a pass phrase.
+ */
+const moveMouseRandomly = () => {
+  for (let i = 0; i < 70; i += 1) {
+    cy.get('main').first().trigger('mousemove', {
+      which: 1,
+      pageX: 500 + (Math.floor((((i % 2) * 2) - 1) * (249 + (Math.random() * 250)))),
+      pageY: 500 + (Math.floor((((i % 2) * 2) - 1) * (249 + (Math.random() * 250)))),
+    });
+  }
+};
+
+const chooseNetwork = (network) => {
```

Comment:
`chooseNetwork` looks to be duplicated in `test/cypress/e2e/login.spec.js`.
 To avoid duplication, `chooseNetwork` could be defined in one "util" file, exported, and imported in both usages.

Reply:
yes, exactly, it is duplicated. ok, will move it to util for now